j2cl-tasks/src/main/java/com/vertispan/j2cl/build/provided/TurbineTask.java

```diff
@@ -0,0 +1,173 @@
+package com.vertispan.j2cl.build.provided;
+
+import com.google.auto.service.AutoService;
+import com.google.common.base.Splitter;
+import com.google.common.collect.ImmutableList;
+import com.google.j2cl.common.SourceUtils;
+import com.google.turbine.diag.TurbineError;
+import com.google.turbine.main.Main;
+import com.google.turbine.options.LanguageVersion;
+import com.google.turbine.options.TurbineOptions;
+import com.vertispan.j2cl.build.task.Config;
+import com.vertispan.j2cl.build.task.Dependency;
+import com.vertispan.j2cl.build.task.Input;
+import com.vertispan.j2cl.build.task.OutputTypes;
+import com.vertispan.j2cl.build.task.Project;
+import com.vertispan.j2cl.build.task.TaskFactory;
+import org.apache.commons.compress.archivers.ArchiveEntry;
+import org.apache.commons.compress.archivers.ArchiveException;
+import org.apache.commons.compress.archivers.ArchiveInputStream;
+import org.apache.commons.compress.archivers.ArchiveStreamFactory;
+import org.apache.commons.io.IOUtils;
+
+import java.io.File;
+import java.io.IOException;
+import java.io.InputStream;
+import java.io.OutputStream;
+import java.nio.file.Files;
+import java.nio.file.Path;
+import java.nio.file.PathMatcher;
+import java.nio.file.Paths;
+import java.util.Collection;
+import java.util.List;
+import java.util.Optional;
+import java.util.Set;
+import java.util.stream.Collectors;
+import java.util.stream.Stream;
+
+import static com.google.common.collect.ImmutableList.toImmutableList;
+
+@AutoService(TaskFactory.class)
+public class TurbineTask extends JavacTask {
+
+    public static final PathMatcher JAVA_SOURCES = withSuffix(".java");
+
+    @Override
+    public String getOutputType() {
+        return OutputTypes.STRIPPED_BYTECODE_HEADERS;
+    }
+
+    @Override
+    public String getTaskName() {
+        return "turbine";
+    }
+
+    @Override
+    public String getVersion() {
+        return "0";
+    }
+
+    @Override
+    public Task resolve(Project project, Config config) {
+        int version = getJavaVersion();
```

Comment:
@niloc132 so we keep it for now and ll remove it in 0.21, right ?

Reply:
Sure - we could probably just fail now too though, your call.

Reply:
let simplify it for now